packages/vite-plugin-cloudflare/playground/dynamic-import-paths/vite.config.ts

```diff
@@ -0,0 +1,17 @@
+import { cloudflare } from "@cloudflare/vite-plugin";
+import { defineConfig } from "vite";
+
+export default defineConfig({
+	environments: {
+		worker: {
+			build: {
+				rollupOptions: {
+					output: {
+						preserveModules: true,
```

Comment:
Will we require users to add such config?

Reply:
No, this is just for the test case to ensure we know the path of the output file. Frameworks generally use a build manifest to know which modules to reference.